self_supervised_pretraining/ssl_train.ipynb

```diff
@@ -5,23 +5,51 @@
    "id": "ceab8707",
    "metadata": {},
    "source": [
+    "# Self-Supervised Learning Using 3D CT Volumes\n",
+    "\n",
     "Copyright (c) MONAI Consortium  \n",
     "Licensed under the Apache License, Version 2.0 (the \"License\");  \n",
     "you may not use this file except in compliance with the License.  \n",
     "You may obtain a copy of the License at  \n",
-    "    http://www.apache.org/licenses/LICENSE-2.0  \n",
+    "&nbsp;&nbsp;&nbsp;&nbsp;http://www.apache.org/licenses/LICENSE-2.0  \n",
     "Unless required by applicable law or agreed to in writing, software  \n",
     "distributed under the License is distributed on an \"AS IS\" BASIS,  \n",
     "WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.  \n",
     "See the License for the specific language governing permissions and  \n",
-    "limitations under the License.  \n",
-    "\n",
+    "limitations under the License."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "95fc8d67",
+   "metadata": {},
+   "source": [
     "##### Note:\n",
     "[TCIA Covid 19](https://wiki.cancerimagingarchive.net/display/Public/CT+Images+in+COVID-19) dataset needs to be downloaded for the execution of this notebook, both datasets need to be downloaded. Downloading data to the `data_root` path defined variable is optimal, if data is downloaded at another location please change the path `data_root` accordingly.\n",
     "\n",
     "Please see the Readme for more details about the dataset. Data splits have already been provided.\n",
     "\n",
-    "##### Import statements"
+    "## Setup Environment"
```

Comment:
A minor change is needed: "## Setup Environment" -> "Setup environment"

Reply:
I am assuming this is implying E --> e for the 'environment'.